crates/tx5-connection/src/webrtc.rs

```diff
@@ -200,15 +213,25 @@ async fn task(
                 }
             }
             Cmd::RecvMessage(msg) => {
+                slow_task = "recv-msg";
                 evt_send.send(WebrtcEvt::Message(msg)).await?;
             }
             Cmd::DataChanOpen => {
+                slow_task = "chan-open";
                 evt_send.send(WebrtcEvt::Ready).await?;
             }
             Cmd::BufferedAmountLow => {
+                slow_task = "buf-low";
                 pend_buffer.clear();
             }
-        }
+        }) {
+            Err(_) => {
+                let err = format!("slow app on webrtc loop task: {slow_task}");
+                tracing::warn!("{err}");
+                Err(Error::other(err))
```

Comment:
This error will now correctly propagate the connection shutdown even if that other slow app timeout is not triggered in conn.rs.